scripts/build_deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+
+declare -a enviroments=(
```

Comment:
If we add many more columns here, we may want to factor out this configuration data into a separate file. See also [URO-142](https://kbase-jira.atlassian.net/browse/URO-142).

An additional column could be a "preview environment" boolean indicating whether `<Placeholder />` components throw errors. It would be good for it to have other effects before adding it as configuration since placeholders are always meant to be ephemeral. See also the comments below on `PlaceholderFactory.tsx`.